### ISSUE TITLE
Optionally require a group to match only once per reference change

### ISSUE THREAD

**Issue by tomasbjerre** [tomasbjerre/simple-stash-commit-checker#17]
Can be used to reject 2 commits that both use same Jira. WIth a message that they should be squashed.

**Comment by tomasbjerre** [comment]
Cant really come up with a simple solution for this. How to present it to committer. Also the implementation should not make the code to complex. Closing it since it should probably be consider as "nice to have" anyway.
